_includes/docs/user-guide/rule-engine-2-0/ce-enrichment-nodes.md

```diff
@@ -437,39 +439,41 @@ Aggregated data will be placed in outgoing message's metadata as a simple key-va
 
 ## tenant details
 
-Adds fields from Tenant to the message body or metadata.
+Enriches the outgoing message with the tenant's details.
 
-* **Select details** - list of the details that need to be added to the message.
-* **Add selected details to** - an option selector that allows the user to choose whether the fetched details should be added to the **Message** or **Metadata**.
+**Configuration**
+
+* **Select details** - list of the details to be fetched from the tenant.
+* **Add selected details to** - controls whether the fetched details should be added to the **_Message_** or **_Metadata_**.
 
 
 ![image](/images/user-guide/rule-engine-2-0/nodes/enrichment-tenant-details-config.png)
 
-Selected details are added into message or message metadata with prefix: **tenant_**.
+Selected details are added into the outgoing message with prefix: <code>tenant_</code>.
 
 **Output connections**
 * **Success:**
   * If tenant's details were successfully fetched.
 * **Failure:**
   * If unexpected error occurs during message processing.
 
-This node functions similarly to the Customer details node. The key difference is that this node retrieves details from the Tenant,
-while the Customer details node retrieves them from the Customer. For a usage example, please refer to the [Customer details node](/docs/user-guide/rule-engine-2-0/enrichment-nodes/#customer-details).
+This node functions similarly to the customer details node. The key difference is that this node fetches details from the tenant,
+while the customer details node fetches them from the customer. For a usage example, please refer to the [customer details node](/docs/user-guide/rule-engine-2-0/enrichment-nodes/#customer-details).
```

Comment:
It will be nice to change the order of the nodes like tenant details and customer details, Since the example exists only for the customer details node. Or create an example for the tenant details node and refer to the example in the customer details node.